docs/source-2.0/guides/smithy-cli/index.rst

```diff
@@ -0,0 +1,20 @@
+.. _smithy-cli:
+
+==============
+The Smithy CLI
+==============
+
+The Smithy CLI enables you to quickly iterate on your Smithy models - we’ve listened to community feedback
+about the gradle plugin, and have added many features in the CLI to simplify your experience.
```

Comment:
This kind of reads like a blog post -- a point in time description of a change, rather than a description of the CLI.